daemon/api.go

```diff
@@ -1679,6 +1679,42 @@ func getUserDetailsFromStore(email string) (string, *osutil.AddUserOptions, erro
 	return v.Username, opts, nil
 }
 
+func createAllKnownSystemUsers(st *state.State, createData *postUserCreateData) Response {
+	var createdUsers []createResponseData
+
+	st.Lock()
+	db := assertstate.DB(st)
+	assertions, err := db.FindMany(asserts.SystemUserType, nil)
```

Comment:
Ah, I guess it's okay to have that functionality all encapsulated inside getUserDetailsFromAssertion below, instead of duplicated here? Realistically we should have very few of those assertions in the typical case.